docs/how-to-guides/admin/deploying-renku.rst

```diff
@@ -259,6 +259,17 @@ If the RenkuLab deployment is relying on an existing GitLab instance, this insta
 #. Set ``Alias`` to <renku-domain>, ``Authorization URL`` to \https://<gitlab-domain>/oauth/authorize, ``Token URL`` to \https://<gitlab-domain>/oauth/token and ``Client ID`` and ``Client Secret`` to the values used in step 1.
 #. Click ``Save``. The new Identity Provider should appear and any user from the stand-alone GitLab instance should be able to login.
 
+Grant a user the GitLab admin role using the GitLab CLI
```

Comment:
I think this should probably either replace or merge with the section above that is now called "GitLab deployed as part of renku"